euphonic/cli/utils.py

```diff
@@ -671,7 +671,7 @@ def __call__(self, parser, args, values, option_string=None):
         section.add_argument(
             '-s', '--save-to', dest='save_to', default=None,
             help='Save resulting plot to a file with this name')
-        section.add_argument('--title', type=str, default='',
+        section.add_argument('--title', type=str, default=None,
```

Comment:
Just to double check `default` bypasses `type`. I think it does, but you've checked this works.

Reply:
I've used this pattern many times before so it _seems_ to work sensibly.